docker-entrypoint.sh

```diff
@@ -26,6 +26,10 @@ fi
 
 if [ "$1" = 'bundle' ]; then
   set -- su-exec metaractor "$@"
+elif [ "$1" = 'rake' ]; then
```

Comment:
This is a fix I picked up with sycamore and ruby 3. Now we make sure to run `rake` correctly.